src/applications/vaos/tests/components/TextareaWidget.unit.spec.js

```diff
@@ -38,12 +36,4 @@ describe('VAOS Component: TextareaWidget', () => {
       'Test',
     );
   });
-
-  it.skip('should call onChange', () => {
```

Comment:
This test was a pain to try to fix. But we don't directly specify the onChange behavior of this widget so I don't think this test is valuable and therefore removed it.